### ISSUE TITLE
Investigate global solution for adapting SVGs to Windows high contrast mode

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1932]
We had around 7 issues pulled up in our audit related to Icon SVGs not adapting to Windows high contrast mode.

See https://css-tricks.com/accessible-svgs-high-contrast-mode/ for more information about this issue.

## Audit info

Type:Expert observation
Principle: Not applicable
Guideline:Not applicable
Success Criteria: Not applicable
Level: Not applicable
Platform: Web
Severity:Medium

### What's the issue?
The incomplete and locked SVG icons are not very visible when Windows high contrast mode is enabled, as the SVG has not adapted to the forced colours

![old-tasks-icon-hc](https://user-images.githubusercontent.com/1811583/132144538-d6db0ac0-221f-45c5-9ecb-fab0d8da8b92.PNG)

### Our recommendation
Adjust the CSS so that SVG icons are adapted to high contrast themes, see [Accessible SVGs in High Contrast Mode](https://css-tricks.com/accessible-svgs-high-contrast-mode/).

**Comment by ActuallyACat** [comment]
Looking a little deeper into this issue... 

We _could_ use the `-ms-high-contrast` media queries to make adjustments when high contrast mode is active in Windows. Something like this:

```
.icon {
  color: inherit; // use whatever colour the consumer defines as the default
  @media screen and (-ms-high-contrast: active) {
    color: #777777; // "middle gray" 
    opacity: 1;
  }
  @media screen and (-ms-high-contrast: white-on-black) {
    color: white;
    opacity: 1;
  }
  @media screen and (-ms-high-contrast: black-on-white) {
    color: black;
    opacity: 1;
  }
}
```

Things that don't quite make this effective as a global solution: 

**We can't prevent someone from reducing the opacity of the icon by reducing the opacity of the parent element**. In almost all scenarios this is what happens as that's our recommended way to change icon colour. When moving into high contrast mode, even if the icon is white on black, if the opacity is low it will be lost. 

**We need to know the background colour the icon is on**. E.g. globally setting icons to black on `black-on-white` theme won't work if the background container is also black.

**Any global approach that is catered to Windows high contrast mode won't help folks who use high contrast mode on other OS.** For example, someone using high contrast mode on a Mac won't benefit from this approach. I've attached some screenshots (apologies, took the photos on a potato)
![contrast](https://user-images.githubusercontent.com/1621182/137409492-6c01b4e0-6988-4078-bb17-87c6d326b520.png)
![contrast2](https://user-images.githubusercontent.com/1621182/137409964-7612fc8b-ef73-4272-86af-44bc24ac5621.png)
